README.md

```diff
@@ -22,18 +22,21 @@ Inherit `rubocop.yml` and add specific settings for each repository like `AllCop
 inherit_from:
   - https://raw.githubusercontent.com/pixta-dev/pixta-rubocop/master/rubocop.yml
 
+inherit_mode:
+  merge:
+    - Include
+    - Exclude
+
 AllCops:
   TargetRubyVersion: TODO
   Include:
-    - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'Gemfile'
     - 'bin/*'
     - 'config/**/*'
     - 'db/**/*'
     - 'deploy/**/*'
-    - 'vendor/**/*'
```

Comment:
[FYI]
Because `vendor/**/*` includes in the [default configs](https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml#L62).